mne_bids/copyfiles.py

```diff
@@ -545,7 +545,7 @@ def copyfile_bti(raw, dest):
 
     Parameters
     ----------
-    raw : instance of Raw
+    raw : mne.io.raw
```

Comment:
There is no `mne.io.raw`, but there is a `mne.io.Raw`

Reply:
> There is no mne.io.raw, but there is a mne.io.Raw

uh, good point - thanks! But then I am wondering why it didn't warn me for far more entries? As you can see in https://github.com/mne-tools/mne-bids/pull/690/commits/88dadc20dec689779f27bf6b71f8742e8d05365d, I had it wrong many more times than just the two that raised a warning :thinking:

Reply:
Maybe it only warns once per object-not-found?

Reply:
possibly :thinking: thanks a lot anyhow, it's passing :-)